src/interface_adapter/vote_out/VoteOutController.java

```diff
@@ -1,4 +1,18 @@
 package interface_adapter.vote_out;
 
+import use_case.vote_out.VoteOutInputBoundary;
+import use_case.vote_out.VoteOutInputData;
+
 public class VoteOutController {
+    final VoteOutInputBoundary userVoteOutUseCaseInteractor;
+
+    public VoteOutController(VoteOutInputBoundary userVoteOutUseCaseInteractor) {
```

Comment:
I feel like the VoteOutInputBoundary should be instantiated (constructed from new) each time instead of passed in as a parameter for the VoteOutController, as to me it seems weird for the View of the program to have a VoteOutInputBoundary on hand to pass in. 

Might be wrong, tell me what you think.

Reply:
i'm leaving that for now because that's what they did in the week 5 coding

Reply:
Sounds good, can you just comment in the class that you're doing that for now because week5 did it?

Reply:
Yeah sure I’ll add a Todo